aiojobs/_job.py

```diff
@@ -50,6 +56,9 @@ def pending(self) -> bool:
     def closed(self) -> bool:
         return self._closed
 
+    def get_name(self) -> str:
+        return self._task.get_name() if self._task else self._name
```

Comment:
```suggestion
    if sys.version_info >= (3, 8):
        def get_name(self) -> str:
            return self._task.get_name() if self._task else self._name
```

Probably want to do this too.

Reply:
Rather than blocking out the whole function I added the check inside the function and did the same thing with the set_task function. This way get and set name on a Job are still available for users on 3.7. The only difference is if the underlying Task name actually gets set.